.github/workflows/notify-plugin-tools.yml

```
@@ -0,0 +1,37 @@
+name: Lapo Docs - Notify Plugin Tools on Code changes
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+jobs:
+  notify:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check for relevant file changes
+        id: check_files
+        run: |
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }})
+          echo "$CHANGED_FILES" > changed_files.txt
+          if grep -i -E "\.(ts|tsx|go)$" changed_files.txt; then
```

Nit: To avoid writing directly on the repo I would change this to:

```suggestion
          echo "$CHANGED_FILES" > /tmp/changed_files.txt
          if grep -i -E "\.(ts|tsx|go)$" /tmp/changed_files.txt; then
```